clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java

```diff
@@ -57,7 +56,7 @@ public static void registerAppInfo(String prefix, String id) {
             AppInfo mBean = new AppInfo();
             ManagementFactory.getPlatformMBeanServer().registerMBean(mBean, name);
         } catch (JMException e) {
-            log.warn("Error registering AppInfo mbean", e);
+            log.warn("Error registering AppInfo mbean: {}", e.getMessage());
```

Comment:
The issue we observed in Streams is, that people see a stack trace and get worried -- they don't pay attention to the log level that is `WARN` -- thus it seems better to never include stack traces on `WARN` to not confuse users. I can revert, too.